### DISCUSSION TITLE
Has anyone gotten RuskDesk server to run under MacOS "Big Sur"?

### DISCUSSION THREAD

**Discussion by TheHippoMan** [rustdesk/rustdesk#2964] [Q&A]
I'm sorry, but my original message turned out to be superfluous. I have deleted it and replaced it with this current message.

I originally couldn't get RustDesk-1.1.9 to function properly as a server on my MacOS "Big Sur" machine. I set up all the proper permissions and even rebooted the Mac, but I still couldn't connect via RustDesk-1.1.9 via my Linux box.  I reported that here.

However, I then tried the following:
(1) Make sure all the necessary permissions are set up properly on MacOS.
(2) Un-install RustDesk
(3) Re-install RustDesk
(4) Launch RustDesk
(5) When asked to set up the permissions, do so again (I was asked about Accessibility, even though that had been previously set properly for RustDesk).

Then, everything worked! I now can connect to my Mac via my Linux box using RustDesk.

So now, all is well. Apparently, RustDesk-1.1.9 needed to be un-installed and re-installed after the initial permissions were set up.

Onward!

**Comment by rustdesk** [comment]
1.1.9 works on >= 10.14 per our test，but do have some users reported permission setup issue #2797，we have no clue so far as we have no env to reproduce the issue.
